src/global.scss

```diff
@@ -37,3 +37,8 @@ body {
   border-bottom-left-radius: 10px;
   border-bottom-right-radius: 10px;
 }
+
+#HdsCookieConsentContainer {
```

Comment:
I'm definitely not a pro with styling, so a bit uncertain if this kind of solution is OK.

Reply:
@japauliina It would be better to use SCSS-class and follow application SCSS naming convention, so in this case better class name would be: `.hds-cookie-consent-container`

Reply:
It is the id reference of the div I needed to modify.

Reply:
It is the modal div id name that is referred here.

Reply:
@japauliina Yes, but still SCSS-classes and namings would conform project's current patterns. But, because apparently HDS CookieConsent does not support custom CSS-classes at all then we can just use this approach.